packages/runtime/src/templates/getApiHandler.ts

```diff
@@ -70,9 +70,7 @@ const makeApiHandler = ({ conf, app, pageRoot, NextServer }: MakeApiHandlerParam
       return bridge
     }
 
-    const {
-      clientContext: { custom: customContext },
-    } = context
+    const customContext = context.clientContext?.custom
```

Comment:
Do we have any way to test this @Skn0tt?
I appreciate we might not have a good testing strategy yet for this.

Reply:
I don't think we have a good testing strategy yet, no. We could probably build something to test background functions using https://webhook.site/, but i'd like to address that in a separate PR/issue.

Reply:
Tracked the need for an E2E test in https://github.com/netlify/next-runtime/issues/2122.

Reply:
Ok - thanks!